Misanthropic-Mistletoes/reactive-nutshell
# Caroline five

# Description

For "Friends" Component:

FriendCard.js was created. Contains code that properly represents the data for a single friend.
FriendsList.js was created. Contains code that renders FriendCard.js. 

Please delete options that are not relevant.


# Testing Instructions

1. commit changes to your current branch
2. git checkout master
3. git fetch --all
4. git pull origin caroline-five
5. run npm start
6. run VSC
7. copy this into your database in your friends array

`"friends": [
    {
      "id": 1,
      "userId": 2,
      "loggedInUserId": 1
    },
    {
      "id": 2,
      "userId": 3,
      "loggedInUserId": 1
    },
    {
      "id": 3,
      "userId": 2,
      "loggedInUserId": 3
    }
  ]`

8. make sure you have someone stored in session storage
9. click "friends" tab in navbar
10. a list of friends should show up (next task is to get only the friends of logged in user to show... currently data for friend.userId populates)

# Checklist:
